Source/WebCore/platform/graphics/PixelBuffer.h

```diff
@@ -77,4 +79,36 @@ class PixelBuffer : public ThreadSafeRefCounted<PixelBuffer> {
     std::span<uint8_t> m_bytes;
 };
 
+// Type to use for functions that use the PixelBuffer data as source during the call, but do not store a reference to the object or modify the data.
```

Comment:
Not necessarily a problem, but this very close to the same type as ConstPixelBufferConversionView.  Might make sense to unify them, but I won't push it.

Reply:
> Also, unless types are relatively trivial, we tend to prefer a file per type, so I think this would be better suited to its own file.

I'll punt this one on the count of: 
- The patch already being written and changing this part is quite much work that often breaks with platforms I cannot build
- The primary feature of the view is that PixelBuffer& gets converted to the view automatically. This would need the caller to include PixelBufferSourceView.h when seemingly they don't use it
- It's relatively trivial type for some subjective view
- It uses the internals of PixelBuffer, so maybe it's easier to maintain, for now, in the same file
- There's no significant downside to having it in the same file

I'd r+ a patch moving it to its own header, though.